asset/js/widget/TermInput.js

```diff
@@ -319,7 +319,12 @@ define(["../notjQuery", "../vendor/Sortable", "BaseInput"], function ($, Sortabl
         }
 
         onButtonClick(event) {
-            if (! this.hasSyntaxError()) {
+            // Exchange terms only when an Enter key is pressed while in term input.
```

Comment:
*while **not** in the term input*

At least that is the purpose of this event handler.